doc/attr.xml

```diff
@@ -1439,6 +1439,42 @@ gap> ChromaticNumber(D);
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="ChromaticNumberFunc">
+<ManSection>
+    <Func Name="ChromaticNumber" Arg="digraph, colouring algorithm" 
+        Label="for a digraph and colouring algorithm"/>
+    <Returns> A non-negative integer.</Returns>
+    <Description>
+      Alterative implementation of the Chromatic Number attribute
+      (See <Ref Attr="ChromaticNumber"/>) that allows the specific algorithm 
+      used to be selected. 
+
+      If <A>digraph</A> is a digraph without loops (see <Ref 
+      Prop="DigraphHasLoops"/>), then <C>ChromaticNumber</C> uses 
+      <A>colouring algorithm</A> to find the least non-negative integer <C>n</C>
```

Comment:
Would it be worth to specify that the algorithm is not given as a string, but as a function?

Reply:
Good point, will make it more clear